ranger-plugin/scripts/enable-ranger-plugin.sh

```diff
@@ -86,6 +119,27 @@ function get_hawq_url() {
   fi
   HAWQ_HOST=${parts[0]}
   HAWQ_PORT=${parts[1]}
+
+  # get hawq standby host
+  # 1. read from command parameter -s
+  # 2. read from hawq-site.xml
+  if [[ -z "$HAWQ_STANDBY_HOST" ]]; then
+    value=''
+    get_hawq_property hawq_standby_address_host
+    local host=$value
+
+    if [[ "$host" == "none" ]]; then
+      HAWQ_STANDBY_HOST=''
+    else
+      HAWQ_STANDBY_HOST=$host
+    fi
+  fi
+
+  # 3. read from user input
+  while [[ -z "$HAWQ_STANDBY_HOST" ]]
```

Comment:
what if there is no standby host at all, why would we force the user to enter the value here ?